apps/3002-checkout/pages/_app.tsx

```diff
@@ -2,8 +2,10 @@ import { useState } from 'react';
 import App from 'next/app';
```

Comment:
`apps/3002-checkout/pages/_app.tsx` should be placed in `src` folder -> `apps/3002-checkout/src/pages/_app.tsx` 

It checks that `packages/nextjs-mf/src/loaders/nextPageMapLoader.ts` supports all nextjs' ways of page definition in `src` folder & `root` folder - https://nextjs.org/docs/advanced-features/src-directory

Reply:
Also, in `apps/3002-checkout` missing `package.json` file.

Reply:
resolved.